examples/teleop-elm/package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "@viamrobotics/sdk-teleop-demo",
-  "version": "0.0.1",
-  "description": "Viam SDK tele-operation demo in Elm",
+  "name": "example",
+  "description": "An example tele-operation application using @viamrobotics/sdk",
   "scripts": {
+    "preinstall": "cd ../.. && make build",
```

Comment:
Is there any reason to do this as `prestart`, instead? The build process isn't very long, and for local dev, this might be more ergonomic

Reply:
the build process isn't that long but still noticeable IMO - I'd prefer to do it less